test/classes.coffee

```diff
@@ -481,6 +483,7 @@ test "`new` works against bare function", ->
 test "#1182: a subclass should be able to set its constructor to an external function", ->
   ctor = ->
     @val = 1
+    return
```

Comment:
Needed to avoid a "derived constructors must return an object or undefined" error.